.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Context**
+* MapFish print version: ?
+* Java version: ?
+* OS: ?
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+
+**How to reproduce**
+
+<!-- Attach a ZIP archive of the application config to the issue. -->
+
+```json
```

Comment:
Maybe not clear to everybody what this is?
something like "print payload (in json)"?